massgov/mayflower
# React/DP-12665: Upgrade Mayflower-react react and node version.

## Description
Upgrades react to version 16.8.1 and node to version 10.15.1. You may need to fix node-sass but only if you receive a warning like `Node Sass could not find a binding for your current environment: OS X 64-bit with Node.js 10.x`. If you do see it, run `npm rebuild node-sass` to fix it.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12665)

## Steps to Test


1. Pull the branch, run `nvm use` to switch to the new version of node and npm. Run `npm install`, then ensure all npm tasks we normally use work.
